apps/fishing-map/features/datasets/datasets.utils.ts

```diff
@@ -247,6 +247,20 @@ export const getActiveDatasetsInActivityDataviews = (
   })
 }
 
+export const getLatestEndDateFromDatasets = (
+  datasets: Dataset[],
+  datasetCategory?: DatasetCategory
+): string => {
+  if (!datasets.length) return ''
```

Comment:
What if there's no visible datasets by default? It'll return `''`, is that expected? Shouldn't it return `DEFAULT_TIME_RANGE.end` in this case to ensure it returns a proper date?

Reply:
Good catch! thanks